lib/LedgerSMB/Scripts/payment.pm

```diff
@@ -1360,6 +1360,7 @@ sub post_payment {
                 push @amount, $temporary_discount;
                 push @cash_account_id, $discount_account_id;
                 push @source, $locale->text('Applied discount');
+                push @memo, "";
```

Comment:
Could you push `undef` instead of an empty string? That way, in the database a NULL value gets stored instead of an empty string.

For future reference: our tests verify that strings without "variables inside" should be using single quotes. E.g. "string with $variable interpolation" vs 'string constant'. Thus the above should have read `''`.

Reply:
the updated commit change to undef